plugins/source/aws/policies/README.md

```diff
@@ -10,6 +10,39 @@ CloudQuery SQL Policies for AWS
 - [AWS Publicly Available](./publicly_available/policy.sql)
 - [AWS Unused Resources](./unused_resources/policy.sql)
 
+## Installing
+
+### Option 1: Downloading via Git
+
+This method is good for local testing if you have git installed. Clone this repository locally:
+
+```bash
+git clone https://github.com/cloudquery/cloudquery.git cloudquery
+```
+
+Check out the tag matching the AWS Source plugin version you are using:
+
+```bash
+cd cloudquery
+git checkout plugins-source-aws-v16.3.0  # Example. Change to match the AWS version in your CloudQuery source config
+```
+
+Change directory into `plugins/source/aws/policies`:
+
+```bash
+cd plugins/source/aws/policies
+```
+
+### Option 2: Downloading via Releases
+
+This is an alternative method to get the correct policies for a given AWS source plugin version, and is a better fit for automation.
+
+1. Go to https://github.com/cloudquery/cloudquery/releases?q=plugins-source-aws&expanded=true
+2. Find the release version that matches the AWS version in your source config
+3. Click on "Assets" to expand the list
+4. Download the Source Code zip file and unzip it.
+5. Change directory into policies: `cd plugins/source/aws/policies`
+
```

Comment:
Maybe we can add `curl -iL https://github.com/cloudquery/cloudquery/releases/download/plugins-source-aws-v16.3.0/aws_linux_amd64.zip -o aws.zip && unzip aws.zip "policies/*"` as a single line command to do this?

Reply:
Ah nice, yeah let's do that